predicators/ground_truth_models/gridworld/nsrts.py

```diff
@@ -0,0 +1,605 @@
+"""Ground-truth NSRTs for the gridworld environment."""
```

Comment:
This file generally looks good. Some minor feedback:
* Remove all unnecessary NSRTs and variables that are currently commented out
* It could be useful to write a comment or two explaining why we have all the extra move and pick operators. This will mostly be useful for us in the future when we try to debug/modify these NSRTs

Reply:
I had some extra NSRTs because we may need those ones once we make the goals in the environment more varied, e.g. via more randomization of initial locations of the various items and stations. So I started to write them because I was gonna use them, but then I wanted to make the first PR more simple and not have the variation in initial state. 

I could leave them in but add a comment explaining why they are left in there?

Reply:
leave it as a comment; that'd be great!